DenisZackharov/teacherss
# Adding bootstrap and using propshaft

### Summary

Remove bootstrap as a gem, adding with css-bundling + js-bundling + propshaft(sprokets alternative)

### How it works

Now for execute the app using `bin/dev` instead of `bin/rails s`

### Test plan

List of steps to manually test introduced functionality:

1. Run server with bin/dev
2. Check that CSS and JS changes is working

# Checklist:


### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, ENV variables, services, scripts, etc.
